cmd/metal-api/internal/service/switch-service.go

```diff
@@ -80,9 +81,11 @@ func (r *switchResource) webService() *restful.WebService {
 		Operation("deleteSwitch").
 		Doc("deletes an switch and returns the deleted entity").
 		Param(ws.PathParameter("id", "identifier of the switch").DataType("string")).
+		Param(ws.PathParameter("force", "if true switch is deleted with no validation").DataType("boolean").DefaultValue("false")).
 		Metadata(restfulspec.KeyOpenAPITags, tags).
 		Writes(v1.SwitchResponse{}).
 		Returns(http.StatusOK, "OK", v1.SwitchResponse{}).
+		Returns(http.StatusBadRequest, "Bad input data", httperrors.HTTPErrorResponse{}).
```

Comment:
FYI: We typically do not include this response type explicitly in the API as it goes for every request. We handle this through the default error.